wolfssl/wolfcrypt/port/Espressif/esp32-crypt.h

```diff
@@ -0,0 +1,123 @@
+/* esp32-crypt.h
+ *
+ * Copyright (C) 2006-2018 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+#ifndef __ESP32_CRYPT_H__
+
+#define __ESP32_CRYPT_H__
+
+#include "esp_types.h"
+#include "esp_log.h"
+
+#ifdef WOLFSSL_ESP32WROOM32_CRYPT_DEBUG
+#undef LOG_LOCAL_LEVEL
+#define LOG_LOCAL_LEVEL ESP_LOG_VERBOSE
+#else
+#undef LOG_LOCAL_LEVEL
+#define LOG_LOCAL_LEVEL ESP_LOG_ERROR
+#endif
+
+#include <freertos/FreeRTOS.h>
```

Comment:
Is it true FreeRTOS will always be available on this platform?

Reply:
Yes, it is.